index.js

```diff
@@ -23,12 +25,32 @@ function SimpleReplace (inputTree, options) {
 SimpleReplace.prototype = Object.create(Filter.prototype);
 SimpleReplace.prototype.constructor = SimpleReplace;
 
-SimpleReplace.prototype.canProcessFile = function (relativePath) {
-  if (this.files.indexOf(relativePath) > -1) {
-    return true;
-  } else {
-    return null;
+SimpleReplace.prototype._processPattern = function(pattern) {
+  if (pattern instanceof RegExp) {
+    return pattern;
+  }
+  var type = typeof pattern;
+  if (type === 'string') {
+    return new Minimatch(pattern);
+  }
+  if (type === 'function') {
+    return pattern;
   }
+  throw new Error('include/exclude patterns can be a RegExp, glob string, or function. You supplied `' + typeof pattern +'`.');
```

Comment:
Can you tweak this error message to reference 'files' instead of 'include/exclude' ?